Objective-C/CBLDatabase.mm

```diff
@@ -496,6 +506,18 @@ - (BOOL) open: (NSError**)outError {
     return YES;
 }
 
+- (void) stopActiveReplications {
+    // Stop all active replications
+    for (CBLReplicator* repl in _activeReplications) {
```

Comment:
This will cause trouble when the replicators being stopped remove themselves from `_activeReplications` -- NSSet will throw an exception because the set is being mutated during enumeration. To work around this you'll need to make a copy of the set and enumerate that.

Reply:
Good catch. Will do